src/consts.ts

```diff
@@ -12,7 +12,8 @@ export const TEAM_NAME_TO_LABEL: { [name: string]: string} = {
     'Cash & Community': 't-c&c',
 };
 
-export const DRY_RUN_SLEEP_MINS = 2;
+export const LINKING_CHECK_RETRIES = 8;
```

Comment:
This is the whole point - instead of waiting for 2 mins, it checks the config every 15 secs 8-times.